rotkehlchen/rotkehlchen.py

```diff
@@ -520,6 +521,20 @@ def add_blockchain_accounts(
         - RemoteError if an external service such as Etherscan is queried and
           there is a problem with its query.
         """
+        if blockchain == SupportedBlockchain.BITCOIN_CASH:
```

Comment:
This should not be in this function. This function has no blockchain specific logic.

1. Can this be in Marshmallow? If yes put it there. Probably it's hard to do though ...
2. This should then be in chain/manager.py in the specific part where BCH is handled.
`elif blockchain == SupportedBlockchain.BITCOIN_CASH:`
3. This also needs a comment to explain what it is you are doing here and why it is needed